src/components/ViewerMessaging/ViewerMessaging.tsx

```diff
@@ -0,0 +1,80 @@
+import BrowserOnly from "@docusaurus/BrowserOnly";
+import React, { useEffect, useState } from "react";
+import MessagingContent from "./MessagingContent";
+import { MessageSchema } from "./schema";
+
+interface ViewerMessagingProps {
+    product: "mobile" | "web";
+    type: "argument" | "command" | "event" | "operation";
+}
+
+export default function ViewerMessagingWrapper(props: ViewerMessagingProps) {
+    // `BrowserOnly` prevents children from being rendered statically during build
+    return <BrowserOnly>{() => <ViewerMessaging {...props} />}</BrowserOnly>;
```

Comment:
Out of curiousity, why don't we want this being rendered statically during build?

Reply:
I wanted to ensure the content is only available dynamically, so we rely on fetching the most up-to-date schema from web/mobile, instead of having to trigger a build manually every time it is updated.

Reply:
Makes sense!